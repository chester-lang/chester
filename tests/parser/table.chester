%{ x.x = 0, y = %{ "x" => 1 } }