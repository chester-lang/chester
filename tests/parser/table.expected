Block(
  statements = Vector(),
  result = Some(
    value = ObjectExpr(
      clauses = Vector(
        ObjectExprClause(
          key = DotCall(
            expr = Identifier(name = "x", meta = None),
            field = Identifier(name = "x", meta = None),
            telescope = Vector(),
            meta = None
          ),
          value = IntegerLiteral(value = 0, meta = None)
        ),
        ObjectExprClause(
          key = Identifier(name = "y", meta = None),
          value = ObjectExpr(
            clauses = Vector(
              ObjectExprClauseOnValue(
                key = StringLiteral(value = "x", meta = None),
                value = IntegerLiteral(value = 1, meta = None)
              )
            ),
            meta = None
          )
        )
      ),
      meta = None
    )
  ),
  meta = None
)